{
	"schema_version": "1.6.0",
	"id": "USN-5508-1",
	"modified": "2022-07-11T14:09:05.514658Z",
	"published": "2022-07-11T14:09:05.514658Z",
	"related": [
		"CVE-2021-46823",
		"UBUNTU-CVE-2021-46823"
	],
	"summary": "python-ldap vulnerability",
	"details": "It was discovered that Python LDAP incorrectly handled certain regular expressions.\nAn remote attacker could possibly use this issue to cause a denial of service.\n",
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:18.04:LTS",
				"name": "python-ldap",
				"purl": "pkg:deb/ubuntu/python-ldap?arch=src?distro=bionic"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.0.0-1ubuntu0.2"
						}
					]
				}
			],
			"versions": [
				"2.4.41-1",
				"2.5.2-1",
				"3.0.0~b4-1",
				"3.0.0-1",
				"3.0.0-1ubuntu0.1"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-ldap",
						"binary_version": "3.0.0-1ubuntu0.2"
					},
					{
						"binary_name": "python-ldap-dbg",
						"binary_version": "3.0.0-1ubuntu0.2"
					},
					{
						"binary_name": "python-pyldap",
						"binary_version": "3.0.0-1ubuntu0.2"
					},
					{
						"binary_name": "python3-ldap",
						"binary_version": "3.0.0-1ubuntu0.2"
					},
					{
						"binary_name": "python3-ldap-dbg",
						"binary_version": "3.0.0-1ubuntu0.2"
					},
					{
						"binary_name": "python3-pyldap",
						"binary_version": "3.0.0-1ubuntu0.2"
					}
				]
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/usn/USN-5508-1.json"
			}
		},
		{
			"package": {
				"ecosystem": "Ubuntu:20.04:LTS",
				"name": "python-ldap",
				"purl": "pkg:deb/ubuntu/python-ldap?arch=src?distro=focal"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.2.0-4ubuntu2.1"
						}
					]
				}
			],
			"versions": [
				"3.1.0-2build1",
				"3.1.0-2build2",
				"3.2.0-3",
				"3.2.0-3ubuntu1",
				"3.2.0-3ubuntu2",
				"3.2.0-3ubuntu3",
				"3.2.0-4ubuntu2"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python3-ldap",
						"binary_version": "3.2.0-4ubuntu2.1"
					},
					{
						"binary_name": "python3-ldap-dbg",
						"binary_version": "3.2.0-4ubuntu2.1"
					},
					{
						"binary_name": "python3-pyldap",
						"binary_version": "3.2.0-4ubuntu2.1"
					}
				]
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/usn/USN-5508-1.json"
			}
		},
		{
			"package": {
				"ecosystem": "Ubuntu:22.04:LTS",
				"name": "python-ldap",
				"purl": "pkg:deb/ubuntu/python-ldap?arch=src?distro=jammy"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.2.0-4ubuntu7.1"
						}
					]
				}
			],
			"versions": [
				"3.2.0-4ubuntu5",
				"3.2.0-4ubuntu6",
				"3.2.0-4ubuntu7"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python3-ldap",
						"binary_version": "3.2.0-4ubuntu7.1"
					},
					{
						"binary_name": "python3-ldap-dbg",
						"binary_version": "3.2.0-4ubuntu7.1"
					},
					{
						"binary_name": "python3-pyldap",
						"binary_version": "3.2.0-4ubuntu7.1"
					}
				]
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/usn/USN-5508-1.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://ubuntu.com/security/notices/USN-5508-1"
		},
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2021-46823"
		}
	]
}
